*** tests/A-003-002-M-ok.out	2014-11-10 10:30:10.000000000 +0000
--- tests/A-003-002-M-ok.outhyp	2014-11-14 02:17:12.000000000 +0000
***************
*** 35,43 ****
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: root:Super User:/home/root
! vader:Darth Vader:/home/vader
! yoda:Master Yoda:/home/yoda
  Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
--- 35,43 ----
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Darth Vader:vader:/home/vader
! Master Yoda:yoda:/home/yoda
! root:Super User:/home/Super User
  Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
